pom.xml

```diff
@@ -241,6 +247,13 @@
             </exclusions>
         </dependency>
 
+        <dependency>
+            <groupId>org.kiwiproject</groupId>
+            <artifactId>kiwi-test</artifactId>
+            <version>${kiwi.test.version}</version>
```

Comment:
To be consistent we should rename this property as `kiwi-test.version`

Reply:
I will do that when I do the kiwiproject version updates

Reply:
OK